### ISSUE TITLE
Should we extract crowdastro.crowd into its own project?

### ISSUE THREAD

**Issue by chengsoonong** [chengsoonong/crowdastro#212]
https://github.com/chengsoonong/crowdastro/tree/master/crowdastro/crowd

This would mean that we may need to tinker a bit with the interface, but it might be a tool that is of broader interest than just applying it to RGZ.

**Comment by MatthewJA** [comment]
Yes, I think we should. Generic crowd algorithms are notably different from astronomical applications. (Then again, it is _crowd_astro...)

If we do that, we should do it after the thesis, and we should also discuss the API.

**Comment by chengsoonong** [comment]
I will explicitly tag issues with thesis if I think you should include it.
